### DISCUSSION TITLE
Can many to many?

### DISCUSSION THREAD

**Discussion by star78152** [wilsonrljr/sysidentpy#49] [Q&A]
Can input and output be many-to-many? I tried for a long time without success...
Can someone help me?

**Comment by star78152** [comment]
Below is my error message:

File "C:\Users\Hao\AppData\Roaming\Python\Python37\site-packages\sysidentpy\base.py", line 125, in regressor_space
    if isinstance(xlag[i], list) and n_inputs > 1:

TypeError: 'int' object is not subscriptable

**Reply by wilsonrljr** [reply]
Hi @star78152 . Currently we only support MISO problems (i.e., multiples inputs and only a single output). I'll take a look at the modification I have to do to handle MIMO problems  for both polyonomials and Neural Nets, but for now you'll be able to use the package for MISO problems only.

**Comment by dj-gauthier** [comment]
@star78152 
I am not sure whether you are still interested in the MIMO problem.  I have started another discussion just now on making a change to the limits of y_lag so it can include the value 0 (the current version requires that it be >0).   If this change is made, then the MIMO problem will be much easier.  Still some coding of multiple MISO problems, but the code should be much more readable if this change to the limits of y_lag is made.  I will eventually add some examples of MIMO problems one way or the other.
